README.md

```diff
@@ -3,37 +3,38 @@ Fill in the text below with your own answers to each question. Add your answers
 
 ## About You
 What are your names, as they appear on the class roster?
-> ...
+> Sara, Khoa
 
 What do you prefer to be called?
-> ...
+> Sara, Khoa
 
 Why are you taking this course?
-> ...
+> I am an intended CS major
 
 What are your concerns about this course?
-> ...
+> I have no background in Operating Systems, so I hope its not going to be a problem 
```

Comment:
Not a problem! I plan to make this course approachable to anyone with a good foundation in CS.